ynh-dev

```diff
@@ -145,6 +147,21 @@ elif [ "$1" = "run" ]; then
 
 
 #####################
+## Halt running VM ##
+#####################
+elif [ "$1" = "halt" ]; then
+    vm=$(vagrant global-status | head -3 | tail -1)
```

Comment:
This code don't work. On my host I have several vagrant (for works and not only yunohost) and in my case it's an other vm which is returned by this line. This implementation could be dangerous.